store/region.js

```diff
@@ -1,16 +1,75 @@
+import * as Permissions from "expo-permissions";
+import * as Location from "expo-location";
+import Constants from "../constants/Constants";
+
 //Action Constants
 const SET_REGION = "SET_REGION";
+const SET_LOCATION_PERMISSIONS = "SET_LOCATION_PERMISSIONS";
+const SET_NEARBY_REGION = "SET_NEARBY_REGION";
 
 //Action Creators
 export const setRegionAction = region => ({ type: SET_REGION, region });
+export const setLocationPermissions = locationPermissions => ({
+  type: SET_LOCATION_PERMISSIONS,
+  locationPermissions
+});
+export const setNearbyRegionAction = region => ({ type: SET_NEARBY_REGION, region });
+
+//Thunks
+export const getLocationPermissionsAsync = () => async dispatch => {
+  try {
+    let { status } = await Permissions.askAsync(Permissions.LOCATION);
+    dispatch(setLocationPermissions(status));
+  } catch (error) {
+      console.log(error);
+      dispatch(threwError(error));
+  };
+};
+
+export const getUserLocationAsync = () => async () => {
```

Comment:
maybe add a comment above this function that says //Helper Function so it's clear that this isn't a Thunk (or is it a Thunk and i'm missing something?)